rotkehlchen/accounting/pot.py

```diff
@@ -378,20 +369,53 @@ def get_prices_for_swap(
             if e.rate_limited is True and out_price is None:
                 raise  # in_price = out_price = None -> notify user
 
-        if out_price is None and in_price is None:
+        if fee_info is not None:
+            try:
+                fee_price = self.get_rate_in_profit_currency(
+                    asset=fee_info[1],
+                    timestamp=timestamp,
+                )
+            except (PriceQueryUnsupportedAsset, RemoteError):
+                fee_price = None
+
+        # Determine whether to use `out_price` or `in_price` for calculations
+        price_to_use: Literal['in', 'out']
```

Comment:
nitpick: Not sure if it's worth using an enum instead of raw strings here.

Reply:
Hm, I would say it will be an overkill since this variable is only for local usage in this function.

Afaik approach with such literals is widely used in our codebase

https://github.com/rotki/rotki/blob/5433c04b162749b3a19a006eabeeeea6d805c6d6/rotkehlchen/premium/sync.py#L223

https://github.com/rotki/rotki/blob/b75036a755ec8e130b51a900f7edfc67f2e0ee3a/rotkehlchen/api/rest.py#L1686

https://github.com/rotki/rotki/blob/58ca7de234284bdc434828f1f2c3e32f1db6f330/rotkehlchen/chain/ethereum/modules/compound/compound.py#L275

etc.

Reply:
Yeah sure but we also have a lot of enums in tthe codebase, even for small functions. And if something is widely used does not always make it correct.

The benefit of an enum is smaller memory footprint. But as you said it's only used in here so you setup an enum for a function. But anyway this is super minor.